bot/exts/moderation/infraction/management.py

```diff
@@ -368,94 +391,96 @@ async def send_infraction_list(
         self,
         ctx: Context,
         embed: discord.Embed,
-        infractions: t.Iterable[t.Dict[str, t.Any]]
+        infractions: t.Iterable[dict[str, t.Any]],
+        prefix: str = "",
+        ignore_fields: tuple[str, ...] = ()
     ) -> None:
         """Send a paginated embed of infractions for the specified user."""
         if not infractions:
             await ctx.send(":warning: No infractions could be found for that query.")
             return
 
-        lines = tuple(
-            self.infraction_to_string(infraction)
-            for infraction in infractions
-        )
+        lines = [self.infraction_to_string(infraction, ignore_fields) for infraction in infractions]
 
         await LinePaginator.paginate(
             lines,
             ctx=ctx,
             embed=embed,
+            prefix=f"{prefix}\n",
             empty=True,
             max_lines=3,
             max_size=1000
         )
 
-    def infraction_to_string(self, infraction: t.Dict[str, t.Any]) -> str:
+    def infraction_to_string(self, infraction: dict[str, t.Any], ignore_fields: tuple[str, ...]) -> str:
         """Convert the infraction object to a string representation."""
-        active = infraction["active"]
-        user = infraction["user"]
         expires_at = infraction["expires_at"]
         inserted_at = infraction["inserted_at"]
         last_applied = infraction["last_applied"]
-        created = time.discord_timestamp(inserted_at)
-        applied = time.discord_timestamp(last_applied)
-        duration_edited = arrow.get(last_applied) > arrow.get(inserted_at)
-        dm_sent = infraction["dm_sent"]
         jump_url = infraction["jump_url"]
 
-        # Format the user string.
-        if user_obj := self.bot.get_user(user["id"]):
-            # The user is in the cache.
-            user_str = messages.format_user(user_obj)
-        else:
-            # Use the user data retrieved from the DB.
-            name = escape_markdown(user['name'])
-            user_str = f"<@{user['id']}> ({name}#{user['discriminator']:04})"
+        title = ""
+        if "id" not in ignore_fields:
+            title = f"**{self.format_infraction_title(infraction)}**"
 
-        if active:
-            remaining = time.until_expiration(expires_at)
-        else:
-            remaining = "Inactive"
+        symbols = []
+        if not infraction["hidden"] and infraction["dm_sent"] is False:
+            symbols.append(constants.Emojis.failmail)
+        if infraction["hidden"]:
+            symbols.append("🕵️")
+        if inserted_at != infraction["last_applied"]:
+            symbols.append("✏️")
```

Comment:
Maybe we should have these as constants at the top of the file, used by both the SYMBOLS_GUIDE and here, so if we want to change a symbol it's just in one place